src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -65,7 +65,7 @@ def edit
         @is_edit_software_action = params[:section] == 'software'
 
         respond_to do |format|
-          format.html { switch_to_webui2 }
+          format.html
```

Comment:
I'm curious: what's the default respond? Is this creating 406 then?

Reply:
The default format is html. So I removed it